airbyte-workers/src/main/java/io/airbyte/workers/temporal/SyncWorkflow.java

```diff
@@ -90,11 +96,36 @@ public StandardSyncOutput run(JobRunConfig jobRunConfig,
                                   StandardSyncInput syncInput) {
       final StandardSyncOutput run = replicationActivity.replicate(jobRunConfig, sourceLauncherConfig, destinationLauncherConfig, syncInput);
 
-      final NormalizationInput normalizationInput = new NormalizationInput()
-          .withDestinationConfiguration(syncInput.getDestinationConfiguration())
-          .withCatalog(run.getOutputCatalog());
-
-      normalizationActivity.normalize(jobRunConfig, destinationLauncherConfig, normalizationInput);
+      if (syncInput.getOperationSequence() != null && !syncInput.getOperationSequence().isEmpty()) {
+        for (StandardSyncOperation standardSyncOperation : syncInput.getOperationSequence()) {
+          if (standardSyncOperation.getOperatorType() == OperatorType.NORMALIZATION) {
+            final NormalizationInput normalizationInput = new NormalizationInput()
+                .withDestinationConfiguration(syncInput.getDestinationConfiguration())
+                .withCatalog(run.getOutputCatalog());
+
+            normalizationActivity.normalize(jobRunConfig, destinationLauncherConfig, normalizationInput);
+          } else if (standardSyncOperation.getOperatorType() == OperatorType.DBT) {
+            final OperatorDbtInput operatorDbtInput = new OperatorDbtInput()
+                .withDestinationConfiguration(syncInput.getDestinationConfiguration())
+                .withOperatorDbt(standardSyncOperation.getOperatorDbt());
+
+            dbtTransformationActivity.run(jobRunConfig, destinationLauncherConfig, operatorDbtInput);
+          } else {
+            final String message = String.format("Unsupported operation type: %s", standardSyncOperation.getOperatorType());
+            LOGGER.error(message);
+            throw new IllegalArgumentException(message);
+          }
+        }
+      } else {
```

Comment:
This is triggering corresponding Temporal Activity depending on operations in the settings of the connection/sync